stdlib/public/runtime/LLVMSupport.cpp

```diff
@@ -28,6 +30,41 @@ void _report_bad_alloc_error(const char *Reason, bool GenCrashDiag) {}
 void __attribute__((__weak__, __visibility__("hidden")))
 report_bad_alloc_error(const char *Reason, bool GenCrashDiag) {}
 #endif
-} // end namespace llvm
+
+// The same for SmallVector: provide the grow_pod implementation (the only
+// SmallVector function which is not inlined) as we don't link LLVM.
+// TODO: This is a hack. This implementaiton is copied from LLVM and has to stay
+// in sync with it.
+
+/// grow_pod - This is an implementation of the grow() method which only works
+/// on POD-like datatypes and is out of line to reduce code duplication.
+void
+#if !defined(_WIN32)
+__attribute__((__weak__, __visibility__("hidden")))
```

Comment:
We need to use the alternatename for providing the default implementation on Windows.

Reply:
Can you give me a patch for that? Or change it after this PR is merged?

Reply:
Sure, I'll get you a patch (need to wait for my current build to finish first).  I'd rather not break the Windows build in the meantime, because the other breakages then pile up.